dev/scripts/nise_ymls/oci/oci_static_data.yml

```diff
@@ -3,24 +3,49 @@ generators:
   - OCIComputeGenerator:
       start_date: {{start_date}}
       end_date: {{end_date}}
+      unit_price: 0.341
+      consumed_quantity: 88583
       currency: USD
       compartment_name: richardprice
       tenant_id: ocid1.tenancy.oc1..EfjkUPxyZSYLvd
+      subscription_id: 25906734
+      tags:
+        tags/free-form-tag: evening
+        tags/orcl-cloud.free-tier-retained: past
+
   - OCIBlockStorageGenerator:
       start_date: {{start_date}}
       end_date: {{end_date}}
+      unit_price: 0.156
+      consumed_quantity: 87278
       currency: USD
       compartment_name: richardprice
       tenant_id: ocid1.tenancy.oc1..EfjkUPxyZSYLvd
+      subscription_id: 25906734
+      tags:
+        tags/new-tags.tarnished-tags: if
+        tags/orcl-cloud.free-tier-retained: role
   - OCINetworkGenerator:
       start_date: {{start_date}}
       end_date: {{end_date}}
+      unit_price: 0.098
+      consumed_quantity: 14634
       currency: USD
       compartment_name: richardprice
       tenant_id: ocid1.tenancy.oc1..EfjkUPxyZSYLvd
+      subscription_id: 25906734
+      tags:
+        tags/free-form-tag: our
   - OCIDatabaseGenerator:
       start_date: {{start_date}}
       end_date: {{end_date}}
```

Comment:
Do these keys need to be removed since they are duplicates now?

Reply:
I think we should keep these to be semi-consistent with other providers.